scripts/jenkins/cloud/ansible/collect-supportconfig.yml

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
could you add the license boilerplate here?

Reply:
done